express_cut/express_service/views.py

```diff
@@ -91,5 +91,58 @@ def users_views(request, pk):
     return Response(status=status.HTTP_500_INTERNAL_SERVER_ERROR)
 
 
+@swagger_auto_schema(methods=['POST'], request_body=DailyScheduleSerializer, responses=swagResp.commonPOSTResponses,
+                     tags=['dailySchedule'], )
+@api_view(['POST'])
+@authentication_classes([SessionAuthentication, BasicAuthentication])
+@permission_classes([IsAuthenticated])
+def schedule_views(request):
+    if request.method == 'POST':
+        data = request.data
+        serializer = DailyScheduleSerializer(data=data)
+        # Checks if the user logged in is a manager & if it's logged in.
+        if not DailySchedulePermissions().POST_permissions(request):
+            return Response(status=status.HTTP_403_FORBIDDEN)
+        # Checks if you have both a date and a stylist id.
+        if not serializer.is_valid():
+            return Response({"message": serializer.errors}, status=status.HTTP_400_BAD_REQUEST)
+        try:
+            schedule = serializer.save()
+            return Response(data={'pk': schedule.pk}, status=status.HTTP_201_CREATED)
+        except:
+            return Response(status=status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+@swagger_auto_schema(methods=['GET', 'DELETE'], responses={**swagResp.commonResponses, **swagResp.getResponse(UserSerializer)},tags=['dailySchedule'],)
+@swagger_auto_schema(methods=['PUT'], request_body=DailyScheduleSerializer, responses=swagResp.commonPOSTResponses,
+                     tags=['dailySchedule'], )
+@api_view(['GET','PUT', 'DELETE'])
+@authentication_classes([SessionAuthentication, BasicAuthentication])
+@permission_classes([IsAuthenticated])
+def schedule_views_put(request, pk):
+    try:
+        schedule = DailySchedule.objects.get(pk=pk)
+    except DailySchedule.DoesNotExist:
+        return Response({'message': 'The daily schedule does not exist.'}, status=status.HTTP_404_NOT_FOUND)
+
+    if request.method == 'GET':
+        serializer = DailyScheduleSerializer(schedule)
+        return Response(serializer.data)
+
+    elif request.method == 'PUT':
+        data = request.data
+        serializer = DailyScheduleSerializer(schedule, data)
+        if serializer.is_valid():
+            serializer.save()
+            return Response(serializer.data)
```

Comment:
Missing HTTP Status code

Reply:
I see it's also missing on other previous implemented function (not related to this PR), please add them.

Reply:
OK, will add.